repondeur/zam_repondeur/views/liasse.py

```diff
@@ -62,6 +62,16 @@ def _do_upload_liasse_xml(context: LectureResource, request: Request) -> Respons
         )
         return
 
+    if errors:
+        if len(errors) == 1:
+            what = "l'amendement"
+        else:
+            what = "les amendements"
+        details = ", ".join(f"{uid} ({cause})" for uid, cause in errors)
+        request.session.flash(
+            Message(cls="warning", text=f"Impossible d'importer {what} {details}.")
```

Comment:
This message will mix French and English, probably not in a really meaningful way for the user. Maybe display the uids only to help resolution and post the cause to rollbar for us?